config.reek

```diff
@@ -35,3 +35,8 @@ LongParameterList:
 # This one just makes sure the Class/Module has a comment. Dumb.
 IrresponsibleModule:
   enabled: false
+
+# Allow modules named "V2", "V1", etc
+UncommunicativeModuleName:
+  accept:
+    - !ruby/regexp /V[0-9]$/
```

Comment:
Should the match only modules named `V1`, or are things like `FooV1` acceptable? If the former, do we need to anchor the regex?